src/index.html

```diff
@@ -7,16 +7,16 @@
   <title>Document</title>
 </head>
 <body>
-  <div class="to-do-list-div">
+  <div class="container">
     <div class="title">
       <p>Today's To Do</p>
     </div>
-    <ul id="to-do-list" class="to-do-list">
-      <li class="list-item">
-        <input type="text" placeholder="Add to your list....">
-      </li>
-    </ul>
-    <button>Clear all completed</button>
+    <div class="to-do-task">
```

Comment:
- [ Required ] Please consider setting the `index` property of a todo to the new length array, what this means is if the array has `5` todos, then the indices of the todos in the array should be as follows.
```JavaScript
[
  {"description":"A","completed":false,"index":1},
  {"description":"B","completed":false,"index":2},
  {"description":"c","completed":false,"index":3},
  {"description":"D","completed":false,"index":4},
  {"description":"E","completed":false,"index":5},
  {"description":"F","completed":false,"index":6}
]
```
This means that the first index should not be zero as is the case in your application, kindly rectify this.

## Screenshot of your App
![Screenshot from 2023-02-24 15-01-50](https://user-images.githubusercontent.com/16392046/221175755-c4573a69-1c2a-4f57-bfa0-827368504b22.png)

- [ Required ] Please consider updating all remaining items indexes so that they represent the current list order and are unique, currently, when I delete a todo from the list all the remaining `indices` are updated with the same value which should not be the case.

![Screenshot from 2023-02-24 15-11-01](https://user-images.githubusercontent.com/16392046/221176817-8d7cf23d-375b-43b0-865d-ae44c1fe3459.png)

## Project requirements
![Screenshot from 2023-02-24 15-16-59](https://user-images.githubusercontent.com/16392046/221177333-cad04592-c33f-48bd-aca0-b31de064ef69.png)